docs/user-guide/global-configurations/git-accounts.md

```diff
@@ -28,36 +28,36 @@ Provide the `URL`. **For example**- [https://github.com](https://github.com) for
 
 ### 4. Authentication type
 
-Here you have to provide the type of authentication required by your version controller. We support three types of authentications. You can choose the one that suits you the best.
+Here provide the type of authentication required by your version controller. Devtron support three types of authentications. you can choose the one that suits you the best.
 
 * **Anonymous**
 
-If you select `Anonymous` then you do not have to provide any username, password/authentication token or SSH key. Just click on `Save` to save your git account provider details.
+If authentication type is set as `Anonymous` then you do not need to provide any username, password/authentication token or SSH key. Just click on `Save` to save the git account provider details. If authentication type is set as `Anonymous`, only public git repository will be accessible.
 
 ![](../../user-guide/global-configurations/images/git-accounts-anonymous.jpg)
 
 * **User Auth**
 
-If you select `User Auth` then you have to provide the `Username` and either of `Password` or `Auth Token` for the authentication of your version controller account. Click on `Save` to save your git account provider details.
+If you select `User Auth` then you have to provide the `Username` and either of `Password` or `Auth Token` for the authentication of your version controller account. Click on `Save` to save the git account provider details.
 
 ![](../../user-guide/global-configurations/images/git-accounts-user-auth.jpg)
 
 * **SSH Key**
 
-If you choose `SSH Key` then you have to provide the `Private SSH Key` corresponding to the public key added in your version controller account. Click on `Save` to save your git account provider details.
+If you choose `SSH Key` then you have to provide the `Private SSH Key` corresponding to the public key added in your version controller account. Click on `Save` to save the git account provider details.
 
 ![](../../user-guide/global-configurations/images/git-accounts-ssh.jpg)
 
 ## Update Git Account
 
-You can update your saved git account settings at any point in time. Just click on the git account which you want to update. Make the required changes and click on `Update` to save you changes.
+You can update your saved git account settings at any point on time. You just need to click on the git account which you want to update. Make the required changes and click on `Update` to save the changes.
```

Comment:
at any point on time > anytime.

Break this into pointers:
1. Click on the git account...
2. Make the required changes...